pkg/ci/plugins/helm/templates/apply.md

```
@@ -14,6 +14,40 @@
+
+{{ else }}
+
+| Field | Value |
+| --- | --- |
+| Operation | `{{ index . "operation" }}` |
+| Wait strategy | `{{ index (index . "wait") "strategy" }}` |
+| Timeout | `{{ index . "timeout" }}` |
+| Chart hooks enabled | `{{ index . "chart_hooks" }}` |
+| Wait for Jobs | `{{ index (index . "wait") "jobs" }}` |
+| On failure | `{{ index . "on_failure" }}` |
+{{ if eq (index . "operation") "install" -}}
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)